src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -123,17 +129,22 @@ class ReportActionContextMenu extends React.Component {
                     );
                 },
             },
-
             {
                 text: this.props.translate('reportActionContextMenu.deleteComment'),
                 icon: Trashcan,
-                shouldShow: false,
-                onPress: () => {},
+                shouldShow: () => this.props.reportAction.actorEmail === this.props.session.email,
+                onPress: () => {
+                    this.setState({isDeleteCommentConfirmModal: true});
+                },
             },
         ];
 
         this.wrapperStyle = getReportActionContextMenuStyles(this.props.isMini);
 
+        this.state = {
+            isDeleteCommentConfirmModal: false,
```

Comment:
I think this should be renamed to one of:

1. `isDeleteCommentConfirmModalVisible`
2. `shouldShowDeleteCommentConfirmModal`